frontend/public/kubevirt/components/vm/vm.jsx

```diff
@@ -64,7 +64,7 @@ const VMRow = ({obj: vm}) => {
         resource={vm}
         resources={[
           getResource(VirtualMachineInstanceModel, {name, namespace, isList: false}),
-          migrationResources,
+          getResource(VirtualMachineInstanceMigrationModel, {namespace}),
```

Comment:
It is still going to result in an array (`isList` is true by default). But that is fine as we filter the migrations array to a single instance in in `menu-actions` - `findVMIMigration`. So there is no difference with this change, or am I missing something ?

Reply:
It didn't result in an array in my testing. I can add the isList: false parameter to make absolutely certain, though.

Reply:
Disregard. You're correct. I've updated the commit message with the actual cause of the problem. Thanks for catching the mistake! =)

Reply:
can you update the commit please?

Reply:
> can you update the commit please?

The commit is still correct. It should be passed as an array and not a standalone object. I just misunderstood the cause of the issue, so the commit message was inaccurate. It has been corrected to provide the correct cause of the problem.